android/com/facebook/buck/android/support/exopackage/ExopackageApplication.java

```diff
@@ -162,4 +162,15 @@ public void onConfigurationChanged(Configuration newConfig) {
       delegate.onConfigurationChanged(newConfig);
     }
   }
+
+  @Override
+  public Object getSystemService(String name) {
+    if (delegate != null) {
+      Object service = delegate.getSystemService(name);
```

Comment:
The name is `@NonNull` in `Context.getSystemService()`. I @rjrjr was referring to the return value for this method.